components/shared-components/navigation/ThemeSwitcher.tsx

```diff
@@ -2,7 +2,7 @@
 
 import Image from 'next/image';
 
-import { sun, moon } from '../../public/assets/index';
+import { sun, moon } from '../../../public/assets/index';
```

Comment:
```js
import { sun, moon } from '@/public/assets/index';
```
Use your paths and you won't have to edit these (as much) when you rework directories